sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusLiveTestBase.cs

```diff
@@ -8,8 +8,7 @@
 
 namespace Azure.Messaging.ServiceBus.Tests
 {
-    [Category(TestCategory.Live)]
-    [Category(TestCategory.DisallowVisualStudioLiveUnitTesting)]
```

Comment:
I'd recommend leaving this; otherwise Visual Studio's Live Unit Testing feature will attempt to run the live tests in real-time for all changes.

Reply:
For the sake of standardization, I will leave this out here. If it enough of an issue, we can add it to the Test Framework.